cw_bitcoin/lib/electrum.dart

```diff
@@ -73,17 +75,9 @@ class ElectrumClient {
     socket = null;
 
     try {
-      if (useSSL == false || (useSSL == null && uri.toString().contains("btc-electrum"))) {
```

Comment:
please add this condition for useSSL for now as it's not guaranteed that `useSSL` is set correctly for all nodes